src/EnergyPlus/Material.cc

```diff
@@ -84,7 +84,7 @@ constexpr std::array<Material::Gas, 10> gases = {
 
 constexpr std::array<std::string_view, (int)EcoRoofCalcMethod::Num> ecoRoofCalcMethodNamesUC = {"SIMPLE", "ADVANCED"};
 
-int GetMaterialNum(EnergyPlusData &state, std::string const &matName)
+int GetMaterialNum(EnergyPlusData const &state, std::string const &matName)
```

Comment:
[src/EnergyPlus/Material.cc:87]:(style),[constParameter],Parameter 'state' can be declared as reference to const